pkg/volume/csi/csi_plugin_test.go

```diff
@@ -129,6 +143,7 @@ func TestPluginGetPluginName(t *testing.T) {
 
 func TestPluginGetVolumeName(t *testing.T) {
 	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.CSIBlockVolume, true)()
+	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.CSIInlineVolume, true)()
```

Comment:
same thing here. would like the keep the test cases where the feature is off so we can validate existing behavior is not broken